docs/production/authentication-methods.md

```diff
@@ -598,6 +598,34 @@ to the root and `engineering` subdomains:
          importing, only the certificate will be displayed (not the private
          key).
 
+
+### Using Authentik as a SAML IdP
+
+1. Make sure you reviewed [this article](https://goauthentik.io/integrations/services/zulip/), which
```

Comment:
Hmm, why this url? Are we expecting them to have a page about integrating with Zulip?

Reply:
Yes. I wrote that one too. Yesterday. We can remove it now and add it later if you want. (mentioned this in the comment btw)

Reply:
This is now live:  https://goauthentik.io/integrations/services/zulip/